### PR TITLE
Remove 2119 language from privacy section.

### LINKED ISSUE #439: RFC2119 keywords in informative sections
The Privacy Considerations section contains RFC2119 keywords, but the section is marked as informative.

### PR DESCRIPTION
**Pull request by adrianba**
Fixes #439.

***
[Preview](https://s3.amazonaws.com/pr-preview/adrianba/browser-payment-api/issue439.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/browser-payment-api/e0cf815...adrianba:4c3b4a7.html)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2690,8 +2689,10 @@
           Exposing user information
         </h2>
         <p>
-          The <a>user agent</a> MUST NOT share information about the user to
-          the web page (such as the shipping address) without user consent.
+          A page might try to call the payment request API to retrieve
+          information about the user. Information about the user (such as the
+          shipping address) should only be shared with the web page with user
```
Comment (marcoscaceres) [inline index.html:2694]: "should" is still an RFC 2119 keyword (particularly as we opted to not capitalize RFC2119 keywords in the spec). 

We can try not to make a normative recommendation here, but I don't think it's particularly harmful to make this section non-normative if we are making a common sense RECOMMENDATION/SHOULD statement here.

Comment (adrianba) [reply index.html:2694]: I think it's crazy to try to make this say should without using the word should and this is a pretty important recommendation that has come up over and over in the group discussion but be my guest if you want to update this with something else.

Comment (domenic) [reply index.html:2694]: It sounds like this section should be normative.

Comment (marcoscaceres) [reply index.html:2694]: Agree. Let's make it normative.

Comment (ianbjacobs) [reply index.html:2694]: Yes, it is intended to be normative.

Ian

**Comment by zkoch** [comment]
Since we merged #561, this is now obsolete, and I am closing as such.

**Pull request closed without merging**